### PR TITLE
Input account and repository Component

### PR DESCRIPTION
**Pull request by geckotang**
gbr
- First, Input `account` and `repository` name to this component.
- Second, Click `Go`.
- Finally, you transit to `#/<account>/<repository>`.

I couldn't come up with a good this component name.

### CODE REVIEW

File: lib/components/form.js [hunk 1]
```diff
@@ -42,21 +42,26 @@ const Form = React.createClass({
...
     })
   },
-  componentDidMount() {
-    let { account, repository } = this.props.params
-    let json = `${endpoints.rawgit}${account}/${repository}/master/issue.json`
-    this.setState({
-      account: account,
-      repository: repository
-    })
+  fetchData(account, repository) {
```
Comment (watilde) [inline lib/components/form.js:45]: how about to add an exception processing?

```
if (!account || !repositry) return
```

Comment (geckotang) [inline lib/components/form.js:45]: It's necessary to check. However, If account or repository was inputted, Where will the user go?

Comment (watilde) [inline lib/components/form.js:45]: Should be next input :smiley:

**Comment by watilde** [comment]
LGTM

**Pull request merged**